services/src/main/scala/cromwell/pipeline/service/ProjectFileService.scala

```diff
@@ -0,0 +1,17 @@
+package cromwell.pipeline.service
+
+import cromwell.pipeline.datastorage.dto.FileContent
+import cromwell.pipeline.womtool.WomTool
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+class ProjectFileService(womTool: WomTool)(implicit executionContext: ExecutionContext) {
+
+  def validateFile(fileContent: FileContent): Future[Either[ValidateError, String]] =
+    Future(womTool.validate(fileContent.content)).map {
+      case Left(value) => Left(ValidateError(value.head)) // thinking about change Left on Future.failed
```

Comment:
`Future[Either[]]` looks good to me.
I'm more concerned about `.head` operation. If there is more than one error, why are you throwing them away?

Reply:
Will fix it, thanks